src/actions/register.js

```diff
@@ -0,0 +1,56 @@
+var { API_URL } = require('../PLEnv');
+var { Action, ThunkAction } = require('./types');
+
+function findByUsername(username: string){
+    return new Promise((resolve, reject) => {
+        fetch(`https://api-staging.powerli.ne/api-public/users/?username=` + username, {
```

Comment:
There is a string value of base url used throughout app.(i.e `API_URL` in `PLEnv.js` file).
You can import it by following code.
```
const { API_URL } = require('PLEnv'); 
```